src/components/IconButton/docs/IconButtonExtendedExample.tsx

```diff
@@ -6,23 +6,36 @@ import { IconButton, Skins } from '../IconButton';
 export class IconButtonExtendedExample extends React.Component {
   render = () => (
     <div>
+      <h3>
+        Override the style params per skin - start with a skin and change what
+        you want - this is common for a component with predefined variations
+      </h3>
       <div>
         <IconButton
           icon={<ShareIcon />}
-          {...this.props}
-          className={classes.root}
+          skin={Skins.Line}
+          className={classes.mixSkinLine}
         />{' '}
         Skin 'line'
       </div>
       <div>
         <IconButton
           icon={<ShareIcon />}
           skin={Skins.Full}
-          {...this.props}
-          className={classes.root}
+          className={classes.mixSkinFull}
         />{' '}
         Skin 'full'
       </div>
+
+      <h3>*Deprecated* - Override the style params in the entire css file</h3>
+      <div>
+        <IconButton
+          icon={<ShareIcon />}
+          skin={Skins.Line}
+          className={classes.mixAll}
+        />{' '}
```

Comment:
Redundant?
```suggestion
        />
```

Reply:
I think this for having some space between the button and the text